packages/nimble-tokens/CONTRIBUTING.md

```diff
@@ -42,7 +42,9 @@ For any token metadata changes (e.g. documentation, code snippets, etc.):
 
 ## Updating icons
 
-1. Export high-quality, optimized SVG files from the `source/icons/Nimble_Iconography.ai` icon source file, by using the **Export for Screens…** workflow within Adobe Illustrator to export SVG files:
+1. Get the latest copy of the icon source file from NI internal Perforce at `\NIComponents\VisualDesign\ProductionAssets\DiagramPaletteAssets_XML\SystemLink\trunk\20.0\source\Nimble_Iconography.ai`. If you don't have access, reach out to Fred Visser or Brandon O'Keefe to request a copy.
```

Comment:
I was assuming/hoping this would be the source of truth 🙄. What would it take to get Brandon off Perforce? Does it matter to any other workflows?

Reply:
I agree that would be better.

At the team meeting yesterday he mentioned that someone else (other designers?) use the p4 copy. He also mentioned that he's still not comfortable with GitHub workflows.

You wanna reach out to him to see if those blockers can be addressed?

Reply:
Yep – After discussing with Brandon, it's clear there are still some use cases for the p4 copy. Let's make this doc change - 👍

One future solution to this mess (i.e. lack of automation in Nimble and p4 required for automation elsewhere) – with the eventual move to Figma, we could declare a the icons contained within a Figma asset library as the single source of truth, and export to all versions from there in a CI pipeline.